### ISSUE TITLE
Add new Mus'haf Madinah font

### ISSUE THREAD

**Issue by Muslim629** [quran/quran_android#1623]
السلام عليكم ورحمة الله و بركاته 
اخواني الكرام كيف حالكم
This is new Mus'haf Madinah program with two fonts Hafs and Warsh, for more visit site: https://nashr.qurancomplex.gov.sa/
![nashr-home](https://user-images.githubusercontent.com/52772116/125031194-9385e100-e08c-11eb-8537-213859041bee.jpg)
![nashr-11](https://user-images.githubusercontent.com/52772116/125031199-94b70e00-e08c-11eb-9740-6e10f9bed884.jpg)
![nashr-3-1](https://user-images.githubusercontent.com/52772116/125031204-9680d180-e08c-11eb-8256-a484482517c1.jpg)
![nashr-select](https://user-images.githubusercontent.com/52772116/125031210-97b1fe80-e08c-11eb-9644-96aeffc9571a.jpg)
![nashr-doc](https://user-images.githubusercontent.com/52772116/125031212-984a9500-e08c-11eb-92f7-6799b4b9933d.jpg)
![nashr-warsh](https://user-images.githubusercontent.com/52772116/125031214-9a145880-e08c-11eb-973a-de279f0c2e8e.jpg)

@ahmedre

**Comment by Muslim629** [comment]
![20210709_082448](https://user-images.githubusercontent.com/52772116/125033125-33dd0500-e08f-11eb-8169-126dfc67933b.jpg)

**Comment by nacer80** [comment]
بارك الله فيك أخي

**Comment by Muslim629** [comment]
وبارك الله

**Comment by ahmedre** [comment]
وعليكم السلام ورحمه الله وبركاته 
I think the tricky thing here is the database they use in this app is password protected, so it's a bit difficult to extract the data from it unless we can figure out the password (or a reliable pattern for getting the data).

**Comment by nacer80** [comment]
السلام عليكم ورحمة الله وبركاته 

All the data used in quran_android app are from KFGQPC.
https://qurancomplex.gov.sa/en/techquran/dev/

وجزاكم الله خيرا

**Comment by ahmedre** [comment]
The one in the screenshots you posted on this issue

**Comment by shakib2018** [comment]
yes the present font displays 'hamza' like this, it would be better if the fonts were changed
![IMG_20220821_152835](https://user-images.githubusercontent.com/86146710/185785078-b0bbd05a-8f2d-4d0b-9ed1-493baa0bd04e.jpg)

**Comment by shakib2018** [comment]
> وعليكم السلام ورحمه الله وبركاته I think the tricky thing here is the database they use in this app is password protected, so it's a bit difficult to extract the data from it unless we can figure out the password (or a reliable pattern for getting the data).

Have you checked this app? https://play.google.com/store/apps/details?id=com.logicdev.quran_reader

**Comment by nacer80** [comment]
السلام عليكم ورحمة الله وبركاته 
I think no difference.
Please check this screenshots from quran_android and quran reader.

Quran_android:
![Screenshot_20220821-112454_Quran](https://user-images.githubusercontent.com/60948528/185786941-6efd6bfd-9644-4a7f-8eb8-65b2247885aa.jpg)
Quran reader:
![Screenshot_20220821-112429_Quran Reader (Dev)](https://user-images.githubusercontent.com/60948528/185786960-7f0435ae-e24c-4310-853c-7d5289064857.jpg)

وجزاكم الله خيرا

**Comment by benomaire** [comment]
@shakib2018 
Are you using Android 5 or older?

**Comment by shakib2018** [comment]
> @shakib2018 Are you using Android 5 or older?

no, Android 8.1; Oppo a5s
I have seen this several places that 'little hamza' showing that way.

**Comment by benomaire** [comment]
@amarlab7 
I think the following might be helpful for you
Go to Settings -> Page Type (Experimental) -> Choose 1439 or 1440

Also you can "enlarge" the Quran text by rotating your phone
